store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -101,15 +72,15 @@ public String sanitize() throws UnsupportedEncodingException {
         // create the builder into which the sanitized email will be written
         final StringBuilder htmlBuilder = new StringBuilder(html.length());
         // create the renderer that will write the sanitized HTML to the builder
-        final HtmlStreamRenderer renderer = HtmlStreamRenderer.create(htmlBuilder,
+        final ExtensibleHtmlStreamRenderer renderer = ExtensibleHtmlStreamRenderer.create(htmlBuilder,
                 Handler.PROPAGATE,
                 // log errors resulting from exceptionally bizarre inputs
                 new Handler<String>() {
 
             public void handle(final String x) {
                 throw new AssertionError(x);
             }
-        });
+        }, LC.zimbra_strict_unclosed_comment_tag.booleanValue(), new HashSet<>(Arrays.asList(LC.zimbra_skip_tags_unclosed_Cdata.value().toString().split(","))));
```

Comment:
Get this in variables and have separate method to get `zimbra_skip_tags_unclosed_Cdata` with exception handling.
```
protected Set<String> getSkipTagsUnclosedCdata() {
}
```

Reply:
added a try-catch block, and logged an exception in the catch block. Should I be throwing it?